---
layout: default
---
<section id="banner-projects">
</section>

<section id="main" class="wrapper banner">
      <header class="major">
        <h2>Projects</h2>
      </header>
      <div class="container">

          <h2>Metabarcoding of insects for accelerated taxonomic discovery</h2>
          <img class="image portrait" src="{{ site.baseurl }}/images/MTsample.jpg" alt="metabarcoding" width="400"/>
          <p>
            This project consists of the optimization of metabarcoding of insects protocols from bulk samples or environmental DNA (eDNA). Marker and PCR primer selection is a key step in metabarcoding. Our research has a strong focus on this aspect, including testing existing primers and designing new ones when necessary, for different mitochondrial genes. In the process we develop pipelines for primer design and metrics for evaluation that can be applied to any taxonomic group.
            <br /><br />
            We also focus on laboratory methods for non-destructive DNA extraction and different sequencing strategies, trying to find the best protocol that effectively extracts DNA while minimizing the effect on the structural integrity of the specimens, so that their morphology can be examined after sequencing by taxonomists. This synergy between metabarcoding and taxonomy can speed up the discovery and description of new species. Also, we take advantage of the great amount of information produced during the metabarcoding pipeline to answer questions related to the ecology or origin of the insect faunas of the habitats from which we sample.
            <br /><br />
            Involved Researchers: <a href="{{ site.baseurl }}/people.html#daniel">Daniel Marquina</a>
          </p>

          <div class="hr-section"></div>
          <hr>

          <h2>Probabilistic modelling of morphological evolution</h2>
          <img class="image portrait" src="{{ site.baseurl }}/images/morphology-model.png" alt="modeling" width="500" />
          <p>
            Morphology represents an important source of data for understanding evolutionary history, particularly for fossil taxa, which remain the only means by which we can directly observe the history of life. Modern statistical phylogenetic inference methods using Bayesian and maximum likelihood frameworks require probabilistic models of evolution, such as the Generalized Time-Reversible (GTR) model of DNA evolution. However, existing probabilistic models of morphological evolution are relatively simple, and fail to capture the complexity of morphological evolution â€“ in the context of phylogenetic inference, their use is analogous to using only the Jukes-Cantor model for phylogenetic inference using nucleotide data. We are thus developing new probabilistic models of morphological evolution to address this gap. In particular, we are modelling correlation between traits, using both discrete-state continuous-time Markov models and multivariate normal models to detect correlation structure in phenomic datasets.
            <br /><br />
            Involved Researchers: <a href="{{ site.baseurl }}/people.html#allison">Allison Hsiang</a>
          </p>

          <div class="hr-section"></div>
          <hr>

          <h2>Automatic organism identification using machine learning</h2>
          <img class="image portrait" src="{{ site.baseurl }}/images/stoneflies.jpg" alt="stoneflies" width="500" />
          <p>
            Machine learning is the best
            <br/> Less work for humans, more work for computers
            <br/> Until the inevitable revolution, that is
            <br/> I, for one, welcome our new robot overlords
            <br/><br/>
            Involved Researchers: <a href="{{ site.baseurl }}/people.html#miroslav">Miroslav Valan</a>, <a href="{{ site.baseurl }}/people.html#allison">Allison Hsiang</a>
          </p>

          <div class="hr-section"></div>
          <hr>

          <h2>Hymenopteran phylogenomics</h2>
          <img class="image portrait" src="{{ site.baseurl }}/images/wasp-phylogeny.jpg" alt="wasp phylogeny" width="500" />
          <p>
            We study gall-wasps to try to understand the genetic basis of their biology. Gall-wasps (or Cynipids) are a family of parasitic wasps that attack oaks and other plants. They are notable for their habit of manipulating their host by forcing it to produce a complex structure where the larvae are fed and protected. However, some cynipids are unable to initiate the process and usually live in the gall of other insects (they are hence termed the "inquilines"). In order to identify genes responsible for gall-induction and clarify its exact mechanism, we compare the genomes of gall-inducers and inquilines. We also work on improving our understanding of both the phylogeny of the group and the life-history of the various species, as this kind of knowledge is essential to the form.
            <br/>
            <br/> Involved Researchers: <a href="{{ site.baseurl }}/people.html#erik">Erik Gobbo</a>
          </p>

          <div class="hr-section"></div>
          <hr>

          <h2>Modeling host-parasite interactions in Hymenoptera</h2>
          <img class="image portrait" src="{{ site.baseurl }}/images/host-parasite.jpg" alt="wasp phylogeny" width="500" />
          <p>
            In collaboration with Michael Landis (Yale University), we are developing a Bayesian approach for inferring the history of association between butterflies and their host plants. We describe the evolution of the repertoire of hosts that a butterfly uses as a continuous-time Markov process of gain and loss of discrete hosts, where gains are influenced by the phylogenetic distance between hosts. This method will allow us to investigate more realistic problems that involve ecological interactions among many taxa.
          <br/>
            <br />Involved Researchers: <a href="{{ site.baseurl }}/people.html#mariana">Mariana Braga</a>
          </p>

          <div class="hr-section"></div>
          <hr>

          <h2>Dipteran taxonomy and morphology</h2>
          <img class="image portrait" src="{{ site.baseurl }}/images/megaselia-hansonix.jpg" alt="Megaselia hansonix" width="500" />
          <p>
            Our projects are centered on exploration of the large, "open-ended" genus Megaselia (Diptera: Phoridae). Large, diverse groups like Megaselia can be excellent tools for investigating ecosystems but the sheer size and diversity of these groups pose a number of logistic challenges. Projects will focus on the identification of Megaselia in Sweden (including the description of species new to science), the phylogeny of Megaselia, streamlining species discovery using massive parallel sequencing (MPS), and recovering life history data of the clade using MPS.
            <br/>
            <br/> Involved Researchers: <a href="{{ site.baseurl }}/people.html#emily">Emily Hartop</a>
          </p>

          <div class="hr-section"></div>
          <hr>

          <h2>Semantic Knowledge Bases for Biodiversity</h2>
          <img class="image portrait" src="{{ site.baseurl }}/images/knowledge-base.png" alt="Semantic knowledge bases" width="500" />
          <p>
          We are building a building a knowledge base (KB) about biodiversity called <a href="http://openbiodiv.net" target="_blank">OpenBiodiv</a>. The KB consists of semantic apps (expert apps) that answer specific questions that biodiversity scientists have, as well as general access services running on top of a semantic graph database. The database is powered by the OpenBiodiv Ontology <http://openbiodiv.net/openbiodivo-20171103> and populated by information extracted from scholarly articles.
          <br/>
          <br/> Involved Researchers: <a href="{{ site.baseurl }}/people.html#viktor">Viktor Senderov</a>
        </p>

        <div class="hr-section"></div>

      </div>

      <div class="hr-section"></div>

</section>
