---
layout: default
---
<section id="banner-software">
</section>

<section id="main" class="wrapper banner">
<!--  <header class="major">
    <h2>Software</h2>
  </header>-->
  <div class="container">
    
    <h2>Software</h2>

    <div class="hr-section"></div>
    
    <img id="mrbayes" class="image portrait" src="http://nbisweden.github.io/MrBayes/img/bayes.jpg" alt="MrBayes" width="200"/>
    
    <p><strong>MrBayes</strong>
      
      <br />
      <a href="http://nbisweden.github.io/MrBayes/">Homepage</a> 
      <br />
      <br />
      
      <a href="http://nbisweden.github.io/MrBayes/">MrBayes</a> was
      one of the first and remains one of the most popular programs
      for Bayesian inference of phylogeny. It covers a large model
      space, supports parallel tempering (Metropolis-coupled MCMC) to
      accelerate convergence, and provides on-the-fly convergence
      diagnostics based on comparisons among two or more independent
      analyses run in parallel. The MrBayes code base is maintained in
      collaboration with the (National Bioinformatics Infrastructure
      Sweden (<a href="https://nbis.se">NBIS</a>) .
     
    </p>
    
    <div class="hr-section"></div>


     
    <img id="revbayes" class="image portrait" src="https://revbayes.github.io/assets/img/gm-tree.svg" alt="RevBayes" width="200"/>

    <p><strong>RevBayes</strong>
      
      <br />
      <a href="https://revbayes.github.io/">Homepage</a> 
      <br />
      <br />
      
      <a href="https://revbayes.github.io/">RevBayes</a> implements Bayesian
      inference for phylogenetic models that are described using a
      dedicated model specification language, Rev, inspired by
      concepts from the field of probabilistic graphical models. This
      allows users to interactively specify their own variants of many
      phylogenetic model components, while combining these components
      with a wide range of more complex model components available
      through flexible function calls in the environment. In this way,
      RevBayes covers a much broader range of models than MrBayes. The
      models can be used in simulation, computationally efficient MCMC
      inference, or path sampling after specifying relevant samplers
      of parameters in the model. RevBayes lead developer is former
      student Sebastian HÃ¶hna, who is now
      at <a href="https://hoehnalab.github.io/">Ludwig Maximilian
      University in Munich. (Link-out)</a>.
      
    </p>
    
    <div class="hr-section"></div>



    <img id="treeppl" class="image portrait" src="{{ site.baseurl }}/images/yggdrasil022.webp" alt="TreePPL" width="200"/>

    <p><strong>TreePPL</strong>
      
      <br />
      <a href="https://treeppl.org/">Homepage</a> 
      <br />
      <br />
      

          Our most recent software project, <a href="https://treeppl.org">TreePPL</a> is based on universal probabilistic programming. The user defines the phylogenetic model using a simple but Turing-complete programming language, and then the system automatically applies appropriate inference algorithms to learn the parameters in the model. This approach gives the user complete freedom in specifying the model. Developing inference algorithms for programmatic model descriptions is challenging but also rewarding, as the same machinery can cover an impressively wide range of different models. Programmatic model descriptions also facilitate the adoption of new inference strategies that have seen little or no use in phylogenetic previously.

      
    </p>
    
    <div class="hr-section"></div>




  </div>
</section>
